packages/solidity/contracts/StakingRewardsStore.sol

```diff
@@ -103,24 +143,24 @@ contract StakingRewardsStore is IStakingRewardsStore, AccessControl, Utils, Time
     }
 
     /**
-     * @dev returns whether the specified pool  is participating in the rewards program
+     * @dev returns whether the specified pool is participating in the rewards program
      *
      * @param program the program data
      *
-     * @return whether the specified pool  is participating in the rewards program
+     * @return whether the specified pool is participating in the rewards program
      */
     function isPoolParticipating(PoolProgram memory program) private view returns (bool) {
```

Comment:
this seems a bit excessive - why not just use the normal `isPoolParticipating` function that receives the pool token?

Reply:
I wanted to save some gas between other calls, but I can remove it

Reply:
the whole thing is very inexpensive so I'd remove it, less code to worry about

Reply:
👍